modules/setting/setting.go

```diff
@@ -749,6 +749,11 @@ please consider changing to GITEA_CUSTOM`)
 		"StampMicro":  time.StampMicro,
 		"StampNano":   time.StampNano,
 	}[Cfg.Section("time").Key("FORMAT").MustString("RFC1123")]
+	// Determine When using custom time format like '2006-01-02 15:04:05'
+	if (TimeFormat == "") {
```

Comment:
No .. this is not assignment to TimeFormat e.g. `TimeFormat = Cfg.Section("time").Key("FORMAT").MustString("RFC1123")